tablet-report-parser/tablet-report-parser.pl

```diff
@@ -635,7 +693,7 @@ sub collect{
 	if (0 == ($self->{UNIQ_TABLETS_ESTIMATE}||=0)){
```

Comment:
That is initialization code for the fist time "sub collect" is called.
In that case, it simply gets set to zero, then compared to zero.
It will always be zero or a positive integer, so no funny false stuff or Real vs. int.